internal/pkg/aws/ecs/service.go

```diff
@@ -70,6 +70,43 @@ func (s *Service) ServiceStatus() ServiceStatus {
 	}
 }
 
+// ServiceConnectAliases returns the ECS Service Connect client aliases for a service.
+func (s *Service) ServiceConnectAliases() []string {
+	if len(s.Deployments) == 0 {
+		return nil
+	}
+	lastDeployment := s.Deployments[0]
+	scConfig := lastDeployment.ServiceConnectConfiguration
+	if scConfig == nil || !aws.BoolValue(scConfig.Enabled) {
+		return nil
+	}
+	var aliases []string
+	for _, service := range scConfig.Services {
+		defaultName := aws.StringValue(service.PortName)
+		if aws.StringValue(service.DiscoveryName) != "" {
+			defaultName = aws.StringValue(service.DiscoveryName)
+		}
```

Comment:
i see - this is so that the `aws` package is agnostic of how we write our CFN template. 

Seems like `if aws.StringValue(service.DiscoveryName) != ""` will always be true because we always write this field: https://github.com/aws/copilot-cli/blob/0d0436a7f074d9c3715bad37e667a71c722a1c64/internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml#L49

Do you think we want a comment about this?

Reply:
Maybe not because in CFN template there's a comment, and here it should be agnostic about how Copilot will use it.